src/Node.php

```diff
@@ -354,7 +354,7 @@ public function jsonSerialize() {
     }
 
     /**
-     * Get the end index of a Node.
+     * Get the end position of a Node.
```

Comment:
I've thought about this, 'position' is not ideal because the language server protocol uses it for line/col and the parser also has `LineCharacterPosition`. There are a ton of references to 'position' so it would be a pain to change, but either index or offset would be better.